assets/espresso/epsilon_x.in

```diff
@@ -0,0 +1,11 @@
+&inputpp
```

Comment:
The names for the blocks may be case-insensitive, however, let's please keep one format in the template files.